packages/react-components/src/button/src/Button.css

```diff
@@ -182,162 +202,163 @@ a.o-ui-button {
+    color: var(--static-white);
+    background-color: var(--foreground-color-5);
 }
 
 /* OUTLINE | ACTIVE | DISABLED */
 .o-ui-button-outline[disabled]:active,
 .o-ui-button-outline[disabled].o-ui-button-active,
 .o-ui-button-outline.o-ui-button-disabled:active,
 .o-ui-button-outline.o-ui-button-disabled.o-ui-button-active {
-    box-shadow: 0 0 0 1px var(--marine-500) inset;
-    color: var(--marine-500);
-    background-color: var(--white);
+    box-shadow: 0 0 0 1px var(--quaternary-border) inset;
+    color: var(--secondary-text);
+    background-color: var(--foreground-color-1);
 }
 
 /* OUTLINE | PRIMARY */
 .o-ui-button-outline.o-ui-button-primary {
-    box-shadow: 0 0 0 1px var(--primary-500) inset;
-    color: var(--primary-500);
+    box-shadow: 0 0 0 1px var(--brand-border) inset;
+    color: var(--brand-text);
 }
 
 /* OUTLINE | PRIMARY | HOVER */
 .o-ui-button-outline.o-ui-button-primary:hover,
 .o-ui-button-outline.o-ui-button-primary.o-ui-button-hover {
-    box-shadow: 0 0 0 1px var(--primary-700) inset;
-    color: var(--primary-700);
-}
-
-/* OUTLINE | PRIMARY | FOCUS */
-.o-ui-button-outline.o-ui-button-primary:focus,
-.o-ui-button-outline.o-ui-button-primary.o-ui-button-focus {
-    box-shadow: 0 0 0 1px var(--primary-700) inset, var(--o-ui-focus-ring);
+    box-shadow: 0 0 0 1px var(--quaternary-border-hover) inset;
+    color: var(--brand-text-hover);
 }
 
 /* OUTLINE | PRIMARY | ACTIVE */
 .o-ui-button-outline.o-ui-button-primary:active,
 .o-ui-button-outline.o-ui-button-primary.o-ui-button-active {
-    box-shadow: 0 0 0 1px var(--primary-500) inset;
-    color: var(--white);
-    background-color: var(--primary-500);
+    box-shadow: 0 0 0 1px var(--brand-border) inset;
+    color: var(--static-white);
+    background-color: var(--foreground-color-brand);
 }
 
 /* OUTLINE | SECONDARY */
 .o-ui-button-outline.o-ui-button-secondary {
-    box-shadow: 0 0 0 1px var(--cloud-200) inset;
-    color: var(--cloud-500);
+    box-shadow: 0 0 0 1px var(--tertiary-border) inset;
+    color: var(--tertiary-text);
 }
 
 /* OUTLINE | SECONDARY | HOVER */
 .o-ui-button-outline.o-ui-button-secondary:hover,
 .o-ui-button-outline.o-ui-button-secondary.o-ui-button-hover {
-    box-shadow: 0 0 0 1px var(--cloud-500) inset;
-}
-
-/* OUTLINE | SECONDARY | FOCUS */
-.o-ui-button-outline.o-ui-button-secondary:focus,
-.o-ui-button-outline.o-ui-button-secondary.o-ui-button-focus {
-    box-shadow: 0 0 0 1px var(--cloud-200) inset, var(--o-ui-focus-ring);
+    box-shadow: 0 0 0 1px var(--tertiary-border-hover) inset;
 }
 
 /* OUTLINE | SECONDARY | ACTIVE */
 .o-ui-button-outline.o-ui-button-secondary:active,
 .o-ui-button-outline.o-ui-button-secondary.o-ui-button-active {
-    box-shadow: 0 0 0 1px var(--cloud-700) inset;
-    color: var(--white);
+    box-shadow: 0 0 0 1px var(--primary-border-active) inset;
+    background-color: var(--foreground-color-2-active);
+    color: var(--static-white);
```

Comment:
Why not "--white" ? Why the "static" prefix?

Reply:
White is modified by the Theme. Static White is a colour that won't change ever.